### PR TITLE
#185218916 Login with Email and Password

### PR DESCRIPTION
**Pull request by frankflej**
## What does this PR do?
This PR consists of a login a simple and fantastic login page
## Description of Task to be completed?
This task adds a feature of allowing users to be able to interact with a very user friendly login page which they will help in the signing to their account
## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ft-login-page-185218916`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run start`

- Check the application's documentation on Storybook: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?
[#185218916](https://www.pivotaltracker.com/story/show/185218916)
## Screenshots (if appropriate)
**Laptop view**
Screen Shot 2015-09-21 at 11 11 17 AM
**Validation view**
Screen Shot 2015-09-25 at 8 02 24 AM

**Mobile phone view**
Screen Shot 2015-09-21 at 11 15 34 AM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="./src/assets/logo.png" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;1,100&display=swap" rel="stylesheet">
```
Comment (nishimweprince) [inline index.html]: Poppins is already imported through the `base.scss` file. Please check on [here](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/ft-login-page-185218916/src/styles/base.scss#L1)

File: src/components/LoginBox.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const LoginBox = () => {
+  return <div>loginbox</div>;
```
Comment (nishimweprince) [inline src/components/LoginBox.jsx]: Is this file still relevant? I believe we might as well remove it.

File: src/styles/authBlueSide.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
+$main_color:#243665 ;
```
Comment (nishimweprince) [inline src/styles/authBlueSide.scss]: The primary color is also already declared in our base.scss file [here](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/ft-login-page-185218916/src/styles/base.scss#L4)

File: src/styles/index.scss [hunk 4]
```diff
@@ -5,6 +5,13 @@
...
 @use 'homeCategoryProductCard';
 @use 'homeTopSeller';
+@use "login";
+@use "authBlueSide";
+@use "loader";
 @import 'tailwindcss/base';
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
+*{
+    font-family: 'Poppins', sans-serif !important;
```
Comment (nishimweprince) [inline src/styles/index.scss:15]: [Here](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/ft-login-page-185218916/src/styles/base.scss#L15) is another reference of this line.

**Review by nishimweprince — COMMENTED** [review]
I have added some suggestions that I believe will make our code more efficient. Please take a look and we see if we can address them before this PR gets merged.

**Review by bergad37 — APPROVED** [review]
Looks great!

**Review by nishimweprince — APPROVED** [review]
Excellent work 🔥

**Review by irakevine — APPROVED** [review]
Well done

**Review by jcharles921 — APPROVED** [review]

**Review by niyontwali — APPROVED** [review]

**Pull request merged**